RetrospectiveExtension.Backend/deploy/env_setup.sh

```diff
@@ -67,19 +68,49 @@
     az webapp create \
         --name "app-${resource_name_suffix}" \
         --resource-group "$resource_group" \
-        --plan "plan-${resource_name_suffix}"
+        --plan "plan-${resource_name_suffix}"\
+        --runtime "DOTNET|5.0"
 
     signalr_connection_string=$( \
         az signalr key list \
             --name "sigr-${resource_name_suffix}" \
             --resource-group "$resource_group" \
-            --query primaryKey \
+            --query primaryConnectionString \
             --output tsv)
 
+    echo "#### Deploying App Settings ####"
     # https://docs.microsoft.com/en-us/cli/azure/webapp/config/appsettings?view=azure-cli-latest#az-webapp-config-appsettings-set
     # Create WebApp AppSettings
     az webapp config appsettings set \
         --resource-group "$resource_group" \
         --name "app-${resource_name_suffix}" \
         --settings Azure__SignalR__ConnectionString=$signalr_connection_string "@dev_certs.json" "@allowed_origins.json"
+
+    #Create Output directory to publish the dotnet project artifacts to be published to the Azure Web Apps Instance
+
+    echo "#### Publishing backend project ####"
+
+    dotnet build
+    # Publish the Dotnet project
+    dotnet publish -c Release 
+
+    cd  bin/Release/net5/publish/
+    # Zip the deployed artifact
+    zip -r "../../../website.zip" .
+
+    cd ../../../..
+   
+    # https://docs.microsoft.com/en-us/cli/azure/webapp?view=azure-cli-latest#az-webapp-deploy
+    # Deploy the web app
+    az webapp deploy --resource-group "$resource_group" \
+        --name "app-${resource_name_suffix}" --src-path "bin/website.zip"
+
+    backend_service_url=$( az webapp show  \
+        --resource-group "$resource_group" \
+        --name "app-${resource_name_suffix}" \
+        --query defaultHostName --output tsv)
+
+    backend_health_check="https://${backend_service_url}/health"
+   
+    echo "backend service deployed at ${backend_service_url}, to check the health of the deployment visit ${backend_health_check}"
```

Comment:
I ran through the full e2e from scratch and it worked great small nit. After seeing the output ("backend service deployed at app-axgold-retro-test1.azurewebsites.net, to check the health of the deployment visit https://app-axgold-retro-test1.azurewebsites.net/health")I would change this slightly to read - "Backend service deployed at app-axgold-retro-test1.azurewebsites.net. To check the health of the deployment visit https://app-axgold-retro-test1.azurewebsites.net/health."